telegram_send.py

```diff
@@ -214,6 +216,9 @@ def send_message(message):
         else:
             for i in images:
                 bot.send_photo(chat_id=chat_id, photo=i)
+    if stickers:
+        for i in stickers:
+            bot.send_sticker(chat_id=chat_id, sticker=i)
```

Comment:
I added the `disable_notification` keyword argument in the last release, so this should be:
`bot.send_sticker(chat_id=chat_id, sticker=i, disable_notification=silent)`